packages/web-components/@sfx/product/src/product.ts

```diff
@@ -0,0 +1,104 @@
+import { customElement, property, html, TemplateResult } from 'lit-element';
+
+import { Base } from '@sfx/base';
+
+@customElement('sfx-product')
+export default class Product extends Base {
+  @property({ type: Object }) product: ProductModel = {};
+
+  constructor() {
+    super();
+    this.updateVariant = this.updateVariant.bind(this);
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+    this.addEventListener('sfx::product_variant_change', this.updateVariant);
+  }
+
+  disconnectedCallback() {
+    super.disconnectedCallback();
+    this.removeEventListener('sfx::product_variant_change', this.updateVariant);
+  }
+
+  updateVariant(e: CustomEvent) {
+    const product = e.detail;
+
+    this.product = {
+      ...this.product,
+      ...product
+    };
+  }
+
+  urlWrap(url: string, children: TemplateResult) {
+    return url ? html`<a href="${ url }">${children}</a>` : children;
+  }
+
+  additionalInfo() {
+    const { product } = this;
+
+    const properties: ProductModel = {
+      name: '',
```

Comment:
If we're moving from a Product having a `name` to having a `title`, this should change too.